homework3/task01/task01.py

```diff
@@ -0,0 +1,26 @@
+from typing import Callable
+
+
+def cache(times: int):
+    cacher = dict()
+
+    def decorator(func: Callable) -> Callable:
+        def wrapper(*args, **kwargs):
+            if func.__name__ not in cacher:
+                cacher[func.__name__] = [func(), times]
+            else:
+                if cacher[func.__name__][1] == 0:
+                    del cacher[func.__name__]
+                    cacher[func.__name__] = [func(), times]
```

Comment:
Why do you add the function to the cache again?

Reply:
Fixed